core/controllers/base.py

```diff
@@ -159,6 +159,9 @@ class BaseHandler(webapp2.RequestHandler):
     # users have agreed to the latest terms.
     REDIRECT_UNFINISHED_SIGNUPS = True
 
+    # What format the get method returns when exception raised, json or html
+    GET_HANDLER_TYPE = feconf.HANDLER_TYPE_HTML
```

Comment:
I was talking with the other tech leads about this and we were slightly worried about other devs relying on this constant for all sorts of other stuff in the future. Do you think it might be more prudent to rename this something like GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_HTML etc. so as to signal more clearly that this constant should not be used for other things?